service_test.go

```diff
@@ -544,6 +578,73 @@ func newServiceMessages(c *Context, path string) Service {
 	return s
 }
 
+type dummyService2 struct {
+	*Context
+	link chan bool
+}
+
+func newDummyService2(c *Context, path string) Service {
+	return &dummyService2{Context: c}
+}
+
+func (ds *dummyService2) ProcessClientRequest(path string, buf []byte) ([]byte, ClientError) {
+	panic("should not be called")
+}
```

Comment:
Well it should not be called and i have to implement the method so... it's a safe measure, if it panics, something's wrong (instead of plain silence).